admin/src/views/Stats.vue

```diff
@@ -540,10 +542,23 @@ export default Vue.extend({
       //   });
     },
     countApplications() {
-      db.collection('DH6').doc('applications').collection('all').get()
-        .then((snap) => {
-          (this as any).applicationCount = snap.size;
-        });
+          (this as any).applicationCount = (this as any).dbref.length;
+    },
+    async countRSVP() {
+      try {
+        const ref = (this as any).dbref;
+        for (let i = 0; i < ref.length; i++) {
+          try {
+            if (ref[i]._.RSVP.coming) {
+              (this as any).rsvp += 1;
+            }
+          } catch (error) {
+            continue;
```

Comment:
Maybe print the error here instead of continue

Reply:
The whole block fails sometimes because some apps in the db don't have RSVP as an array yet so I put continue there to make sure the loop doesn't exit on an app that doesn't have it yet

Reply:
Think that can be mitigated with just if(ref[i]._.RSVP && ref[i]._.RSVP.coming), then the try catch isn't even needed. But at any rate it's probably fine as is, it's just the lint doesn't like 'continue'

Reply:
That worked and I updated it, thanks :)

Reply:
Great!